scripts/attempt-backport.js

```diff
@@ -129,7 +130,23 @@ function attemptBackport (options, version, isLTS, cb) {
     const _cb = cb
     setImmediate(() => {
       options.logger.debug(`backport to ${version} failed`)
-      if (!isLTS) updatePrWithLabels(options, [`dont-land-on-v${version}.x`])
+
+      if (!isLTS) {
+        fetchExistingThenUpdatePr(options, [`dont-land-on-v${version}.x`])
+      } else {
+        getBotPrLabels(options, (err, ourLabels) => {
+          if (err) {
+            options.logger.error(err, 'Error fetching existing bot labels')
+            return
+          }
+
+          const label = `lts-watch-v${version}.x`
+          if (ourLabels.includes(label)) return
```

Comment:
You'll need to negate this. Before I started nagging about `Array.includes()` this line was

```js
if (ourLabels.indexOf(label) === -1) return
```

that would translate to `!ourLabels.includes(label)`